students/dominic_martin/lesson04/mailroom.py

```diff
@@ -0,0 +1,71 @@
+#!usr/bin/env Python3
+
+donation_list = {'name': ('Ted Bayer', 'Panfila Alvarez', 'JR Reid', 'Simon Laplace', 'Jennifer Meyers'), 'donation': (500, 700, 330, 440, 800)}
+donation_list1 = {'Ted Bayer': 500, 'Panfila Alvarez': 700, 'JR Reid': 330, 'Simon Laplace': 440, 'Jennifer Meyers': 800}
+
+
+a = int(input("What would you like to do?\n(1) - Send a Thank You?\n(2) - Create a Report?\n(3) - Send Letters to Everyone?\n(4) - Quit?\nEnter your response:"))
+
+
+def choices(a):
+	global donation_list
+	switch_dict = {1:thanks, 2:genReport, 3:writeLetters, 4:quitAll}
+	switch_dict.get(a, quitAll)()	
+	print()
+
+def thanks():
+	global donation_list
+	b = int(input("Who would you like to send a thanks to?\n(1) - Ted Bayer\n(2) - Panfila Alvarez\n(3) - JR Reid\n(4) - Simon Lapalce\n(5) - Jennifer Meyers\n(6) - Enter a new name\nEnter your choice:"))	
+	nameIdx = (int(b - 1))
+	if nameIdx < 5:
+		donamt(nameIdx)
+	if nameIdx == 5:
+		newDonor(nameIdx)
+
+def newDonor(nameIdx):
```

Comment:
Function names should be snake_case per PEP8: def new_donor(name_id_x) for example